staramr/results/AMRDetectionSummary.py

```diff
@@ -76,3 +109,31 @@ def create_summary(self, include_negatives=False):
             df = df.merge(ds, on='Isolate ID', how='left').fillna(value={'Plasmid Genes': 'None'})
 
         return df.sort_index()
+
+    def create_detailed_summary(self, include_negatives=True):
+        df = self._resfinder_dataframe
+        df['Data Type']='Resistance'
+        ds = self._plasmidfinder_dataframe
+
+        column_names = ['Gene', 'Predicted Phenotype','%Identity', '%Overlap', 'HSP Length/Total Length','Contig', 'Start', 'End', 'Accession', 'Data Type']
+
+        ds = ds.reindex(columns=column_names)
```

Comment:
This line should likely be after `if ds is not None` since if it was `None` it was raise an exception.

Reply:
I decided to remove that completely since that reindex method would be best called in the end of the `create_detailed_summary` method and rearranged my logic in the `_include_detailed_negatives` method where it would include empty rows of plasmids despite not having plasmid frame equal to`None`